docs/custom-apps.md

```diff
@@ -307,8 +307,6 @@ We want to make it easier to deploy **Zulip Readers** on
 Zulip hardware.  The following document talks about how we want to enable this
 from a code structuring standpoint:
 
-[Writing contrib bots](https://github.com/zulip/zulip/blob/master/contrib_bots/bots/readme.md)
-
```

Comment:
Hey. This can be ignored, won't be in the final merge.

Full explanation: This was a change in the last commit meant not to be merged. Did this to make sure the Travis build would pass on the PR(this link isn't in master and would fail) and thus, ensure that there are hopefully no other mistakes in the migration. Sorry for the confusion.